core/Pos/Binary/Class/Core.hs

```diff
@@ -46,10 +52,47 @@ import qualified GHC.Generics               as G
 import           Serokell.Data.Memory.Units (Byte, fromBytes, toBytes)
 import           Universum
 
+
+----------------------------------------------------------------------------
+-- Class definition
+----------------------------------------------------------------------------
+
+-- | Extra configuration for decoder.
+data DecoderConfig = DecoderConfig
+    { _dcNocheck :: Bool
```

Comment:
I won't ask to add docs for `_dcNocheck` in this PR (because FOUR GRE²N CHECKS), but if you could do it locally and include in some unrelated PR, it'd be nice.

Reply:
upd: since I'm told by @volhovM that there's a conflict now, you can add docs :trollface: and maybe even rename it to `_dcNoCheck` for consistency with `decodeFullNoCheck`

Reply:
Done.